server.js

```diff
@@ -134,6 +135,8 @@ async function work(body) {
     skipCollaboratorPR: false,
   };
 
+  repoConfig = environment.checkEnvironmentForConfig(repoConfig);
+
```

Comment:
You should be able to replace the entire logic in this diff with the following code:

```js
if (process.env.MENTION_BOT_CONFIG) {
  try {
    repoConfig = {
      ...repoConfig,
      ...JSON.parse(process.env.MENTION_BOT_CONFIG)
    };
  } catch(e) {
    console.error(
      'Error attempting to read the config from the environment variable ' +
      ' MENTION_BOT_CONFIG'
    );
    console.error(e);
  }
}
```